3-Authorization-II/protect-web-api/AppCreationScripts/README_STEPS.md

```diff
@@ -0,0 +1,471 @@
+
+1. If you have not made a Java SE Web App in your subscription for this project, select the option, `create`, be pressing enter
+1. When prompted for an OS option, select `Linux` by pressing enter.
+1. When prompted for a javaVersion, select `Java 11` by entering the associated number for it
+1. When prompted for a Pricing Tier, select `B1` or any other option for your pricing tier by entering the associated number for it
+1. Finally, press enter on the last prompt to confirm your selections.
+1. Open your pom.xml, it should now be updated with build information for deploying to Azure App Services
+
+#### Step 2: Publish your files (java-spring-webapi-auth)
+
+1. In the terminal window, enter the following command:
+
+        mvn clean package azure-webapp:deploy
+
+1. The terminal log should display a successful build after a few minutes stating that the deployment was successful
+
+> :note: The maven plugin uses information from your subscription to deploy to Azure App Services. Deployment is subject to the limtations of your subscription. Modify your pom.xml to meet your subscription limits.
+
+
+### Deploying web app to Azure App Services
+
+There is one web app in this sample. To deploy it to **Azure App Services**, you'll need to:
+
+- create an **Azure App Service**,
+- update its client(s) to call the website instead of the local environment and,
+- publish the projects to the **App Services**
+#### Publishing using Maven
+
+#### Step 1: Set up the configuration for the azure webapp maven plugin.
+
+1. Open a terminal window in the base directory of your Java Spring project and enter the following command:
+
+        mvn com.microsoft.azure:azure-webapp-maven-plugin:2.5.0:config
+
+1. If you have not made a Java SE Web App in your subscription for this project, select the option, `create`, be pressing enter
+1. When prompted for an OS option, select `Linux` by pressing enter.
+1. When prompted for a javaVersion, select `Java 11` by entering the associated number for it
+1. When prompted for a Pricing Tier, select `B1` or any other option for your pricing tier by entering the associated number for it
+1. Finally, press enter on the last prompt to confirm your selections.
+1. Go to your pom.xml, it should now be updated with build information for deploying to Azure App Services
+
+#### Step 2: Publish your files (java-spring-webapp-auth)
+
+1. In the terminal window in the base directory of your Java Spring project and enter the following command:
+
+        mvn clean package azure-webapp:deploy
+
+1. The terminal log should display a successful build after a few minutes stating that the deployment was successful
+
+> :note: The maven plugin uses information from the subscription Id you provide to deploy to Azure App Services. Deployment is subject to the limtations of your subscription. Modify your pom.xml to meet your subscription limits.
+
+#### Publishing using Visual Studio
```

Comment:
Can we reorder this section to follow the same as in web API?

In web API steps, you have: 
-Publishing using Visual studio
-Publishing using Maven


In web app steps, you have
-Publishing using Maven
-Publishing using Visual studio